### DISCUSSION TITLE
NFC пропуск на IOS

### DISCUSSION THREAD

**Discussion by muteec** [0niel/university-app#343] [Идеи]
Тема слишком очевидная для того чтобы заучить впервые, но я в этом разделе её не нашел.

**Comment by 0niel** [comment]
Привет. NFC пропуска на iOS нет и, скорее всего, никогда не будет. Это связано с политикой компании Apple. Ограничения платформы, мы ничего не можем с этим сделать.

**Reply by muteec** [reply]
Жаль, спасибо за столь оперативный ответ. Удачи
